garrysmod/gamemodes/terrortown/gamemode/vgui/sb_team.lua

```diff
@@ -18,6 +18,83 @@ local function CompareScore(pa, pb)
    return a:Frags() > b:Frags()
 end
 
+local function CompareAlpha(pa, pb)
+   if not IsValid(pa) then return false end
+   if not IsValid(pb) then return true end
+
+   local a = pa:GetPlayer()
+   local b = pb:GetPlayer()

```

Comment:
Might want to use some more descriptive variable names for the rows and players. It took me longer to figure out the purpose of this code than if they had been properly labelled.